src/components/SearchForm/SearchForm.tsx

```diff
@@ -83,6 +83,7 @@ type SearchFormProps = {
   placeholderText: string;
   handleSubmit: ({ searchTerm }: { searchTerm: string }) => void;
   analyticsSearchSource: SearchSourceValueType;
+  placeholder?: string;
```

Comment:
what's this for?

Reply:
yep, we added placeholdertext and placeholder at the same time independantly, definitely only need one of them